rosdep/python.yaml

```
@@ -6349,6 +6357,12 @@ python3-dubins-pip:
   ubuntu:
     pip:
       packages: [dubins]
+python3-elasticsearch:
+  debian: [python3-elasticsearch]
+  fedora: [python3-elasticsearch]
+  opensuse: [python3-elasticsearch]
+  rhel: [python3-elasticsearch]
```

Looks like this is only available for RHEL 8.

```suggestion
  rhel:
    '*': [python3-elasticsearch]
    '7': null
```